### ISSUE TITLE
Facet Navigation and Screen Reader(s) in Blacklight 8

### ISSUE THREAD

**Issue by VanessaVenti** [projectblacklight/blacklight#3748]
**VoiceOver**: In Blacklight 8, when expanded, the first item (e.g., creator, date) reads the name of the filter, not the count associated with the filter. Each filter should include the number of items related to the filter when reading out to screenreaders.

**WCAG Criteria**: 1.1.1 Non-text Content

**Audit Method**: Screenreader testing

**Action**: Ensure all items and their respective counts are read correctly
